packages/salesforcedx-vscode-apex-replay-debugger/README.md

```diff
@@ -75,36 +72,53 @@ TIP: If your log file is part of your Salesforce DX project, you don’t need to
 
 ## Debug Your Code
 
-Set breakpoints, then replay your debug log and inspect your variables’ values.
+Set breakpoints or checkpoints, then replay your debug log and inspect your variables’ values.
 
 1. To set line breakpoints, open a `.cls` or `.trigger` file and click the column to the left of the line numbers.
+1. For more information than line breakpoints provide, add checkpoints. You can set up to five checkpoints to get heap dumps when lines of code run. All local variables, static variables, and trigger context variables have better information at checkpoints. Trigger context variables don’t exist in logs and are available only at checkpoint locations.  
+In Visual Studio Code, a checkpoint is a type of breakpoint. Set up and upload your checkpoints, then start an Apex Replay Debugger session.  
+    1. Set checkpoints on up to five lines in Apex classes or triggers.  
+        1. Click the line of code where you want to set the checkpoint.  
+        1. Open the command palette (press Cmd+Shift+P on macOS or Ctrl+Shift+P on Windows or Linux).  
+        1. Run **SFDX: Toggle Checkpoint**.  
+
+        - Or, right-click in the gutter to the left of the line numbers, select **Add Conditional Breakpoint** | **Expression**, and set the expression to `Checkpoint`.  
+
+        - Or, to convert an existing breakpoint into a checkpoint, right-click the breakpoint, and select **Edit Breakpoint** | **Expression**. Set the expression to `Checkpoint`.  
+
+    1. To upload your checkpoints to your org so that heap dump information can be collected, open the command palette, and run **SFDX: Update Checkpoints in Org**.
```

Comment:
editing nit: "To upload your checkpoints to your org to collect heap dump info, open the..."